ui/tips-renderer/src/test/kotlin/org/cru/godtools/tool/tips/ui/TipBottomSheetDialogFragmentDataModelTest.kt

```diff
@@ -0,0 +1,93 @@
+package org.cru.godtools.tool.tips.ui
+
+import androidx.arch.core.executor.testing.InstantTaskExecutorRule
+import app.cash.turbine.test
+import io.mockk.every
+import io.mockk.mockk
+import io.mockk.verifyAll
+import java.util.Locale
+import kotlinx.coroutines.Dispatchers
+import kotlinx.coroutines.ExperimentalCoroutinesApi
+import kotlinx.coroutines.flow.MutableSharedFlow
+import kotlinx.coroutines.test.TestScope
+import kotlinx.coroutines.test.UnconfinedTestDispatcher
+import kotlinx.coroutines.test.runTest
+import kotlinx.coroutines.test.setMain
+import org.cru.godtools.base.tool.service.ManifestManager
+import org.cru.godtools.db.repository.TrainingTipsRepository
+import org.junit.Assert.assertFalse
+import org.junit.Assert.assertTrue
+import org.junit.Before
+import org.junit.Rule
+import org.junit.Test
+
+@OptIn(ExperimentalCoroutinesApi::class)
+class TipBottomSheetDialogFragmentDataModelTest {
+    private companion object {
+        private const val TOOL = "tool"
+        private const val TIPID1 = "tipId1"
```

Comment:
Is this normal naming convention in Kotlin over `TIP_ID_1`?

Reply:
nope, I can rename it. it's only used internally in this test